node-red/Dockerfile

```diff
@@ -30,5 +30,4 @@ COPY settings.js /usr/src/node-red/.node-red/
 # change the startup command to be sure to use our settings.
 CMD ["npm", "start", "--", "--userDir", "/data", "--settings", "/usr/src/node-red/.node-red/settings.js"]
 
-# end of file
-
+# end of file
```

Comment:
OK to remove line 34, but 33 must end with a newline.

Reply:
It's done.